packages/expo-yarn-workspaces/bin/expo-yarn-workspaces.js

```diff
@@ -10,17 +10,29 @@ const spawnSync = require('../common/cross-spawn-sync');
 let argv = minimist(process.argv.slice(2));
 switch (argv._[0]) {
   case 'check-workspace-dependencies':
-    spawnSync('node', [path.join(__dirname, 'check-workspace-dependencies.js')], {
+    _spawnSubprocess('node', [path.join(__dirname, 'check-workspace-dependencies.js')], {
       stdio: 'inherit',
     });
     break;
 
   case 'postinstall':
-    spawnSync('node', [path.join(__dirname, 'symlink-necessary-packages.js')], {
+    _spawnSubprocess('node', [path.join(__dirname, 'symlink-necessary-packages.js')], {
       stdio: 'inherit',
     });
-    spawnSync('node', [path.join(__dirname, 'make-entry-module.js')], {
+    _spawnSubprocess('node', [path.join(__dirname, 'make-entry-module.js')], {
       stdio: 'inherit',
     });
     break;
 }
+
+function _spawnSubprocess(...args) {
+  let result = spawnSync(...args);
```

Comment:
```suggestion
  const result = spawnSync(...args);
```

Reply:
The rest of this package uses `let` -- leaving it as-is.